lib/crystalball/rspec/runner.rb

```diff
@@ -95,6 +95,35 @@ def run_specs(example_groups)
       def check_examples_limit(example_groups)
         self.class.check_limit(@configuration.output_stream) { @world.example_count(example_groups) }
       end
+
+      def setup(err, out)
+        configure(err, out)
+        @configuration.load_spec_files
+
+        # Since prediction compacting is disabled we need to remove filtering
+        # in cases like './spec/foo.rb ./spec/foo.rb[1:1:2]'
+        each_file_without_id do |file|
+          path = ::RSpec::Core::Metadata.relative_path(File.expand_path(file))
+          @configuration.filter_manager.inclusions[:ids].delete(path)
+        end
+
+        @world.announce_filters
+      end
+
+      def configure(err, out)
```

Comment:
Why do you need it here? Isn't it just a copy of https://github.com/rspec/rspec-core/blob/master/lib/rspec/core/runner.rb#L124 ?

Reply:
Only after `3.7`, in previous versions it is part of the setup.

Reply:
I don't like to keep it here. What if we require users to use rspec 3.7 or higher?

Reply:
I think keeping it here is a minor inconvenience compared to requiring users to use the latest RSpec. Those lines have literally been the same for at least [four years](https://github.com/rspec/rspec-core/blob/8ae96334df032f80548a2583544ee44653f7ba0a/lib/rspec/core/command_line.rb#L21-L23).